README.md

```diff
@@ -81,7 +81,7 @@ Everyone who contributes to this book, no matter how small or big their contribu
 Long-term contributors of the project are considered part of the core contributors groups who take on various leadership roles in the project, as described in the [Ways of Working document](https://github.com/the-turing-way/the-turing-way/blob/main/ways_of_working.md).
 
 The project is coordinated by the co-lead investigators **Kirstie Whitaker** (founder) and **Malvika Sharan**, and hosted at [The Alan Turing Institute](https://www.turing.ac.uk/).
-**Anne Lee Steele** is the Community Manager of The Turing Way since March 2022. **Alexandra Araujo Alvarez** was the Research Project Manager between February 2023 and April 2024.
+**Anne Lee Steele** is the Community Manager of The Turing Way since March 2022. **Léllé Demertzi** has been the Research Project Manager since October 2024. **Alexandra Araujo Alvarez** was the Research Project Manager between February 2023 and April 2024.
```

Comment:
```suggestion
**Anne Lee Steele** is the Community Manager of The Turing Way.
She joined in the role in March 2022.
**Léllé Demertzi** has been the community's Research Project Manager since October 2024.
**Alexandra Araujo Alvarez** was the Research Project Manager between February 2023 and April 2024.
```

Tiny style update! Really great to have these names updated 🌟

Reply:
Thanks so much @KirstieJane ! 🙏